src/test/java/org/fundacionjala/pivotal/steps/EditProjectSteps.java

```diff
@@ -0,0 +1,65 @@
+package org.fundacionjala.pivotal.steps;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.fundacionjala.pivotal.common.Base;
+import org.fundacionjala.pivotal.objects.DashBoard;
+import org.fundacionjala.pivotal.objects.DescriptionValues;
+import org.fundacionjala.pivotal.objects.EditProject;
+import org.testng.Assert;
+import java.util.Map;
+
+/**
+ * Class por Edit Project Steps.
+ */
+public class EditProjectSteps extends Base {
+
+    /**
+     * Init editProject objects.
+     */
+    private EditProject editProject = new EditProject();
+
+    /**
+     * Init DashBoard objects.
+     */
+    private DashBoard dashBoard = new DashBoard();
+
+    /**
+     * Init Map of DescriptionValues objects.
+     */
+    private Map<DescriptionValues, Object> descriptions;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "descriptions" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)